pyvista/plotting/tools.py

```diff
@@ -8,13 +8,18 @@
 
 import pyvista
 from pyvista import _vtk
-from .theme import parse_color, rcParams
+from .colors import string_to_rgb
 
 
+FONT_KEYS = {'arial': _vtk.VTK_ARIAL,
```

Comment:
maybe an Enum for FONT_KEYS? Whoud allow type-hinting (in principal Literal would also be a possible type-hint, but its imho a little bit harder to keep u to date). Of course thats only relevant, if we havve a method set_font() or change the font-dict in a small class (another suggestion I would consider)

Reply:
Done.